### PR TITLE
Use unique

### PR DESCRIPTION
**Pull request by AyaMorisawa**

### CODE REVIEW

File: src/client/app/desktop/views/components/post-form.vue [hunk 1]
```diff
@@ -397,7 +397,7 @@ export default Vue.extend({
 			if (this.text && this.text != '') {
 				const hashtags = parse(this.text).filter(x => x.type == 'hashtag').map(x => x.hashtag);
 				const history = JSON.parse(localStorage.getItem('hashtags') || '[]') as string[];
-				localStorage.setItem('hashtags', JSON.stringify(hashtags.concat(history).reduce((a, c) => a.includes(c) ? a : [...a, c], [])));
+				localStorage.setItem('hashtags', JSON.stringify(unique(hashtags.concat(hashtags))));
```
Comment (syuilo) [inline src/client/app/desktop/views/components/post-form.vue]: `hashtags.concat(history)` ?

Comment (AyaMorisawa) [reply src/client/app/desktop/views/components/post-form.vue]: :cry:

**Comment by AyaMorisawa** [comment]
Updated

**Comment by syuilo** [comment]
👍

**Pull request merged**